src/main/java/com/health/config/SecurityConfig.java

```diff
@@ -52,7 +52,8 @@ public static BCryptPasswordEncoder passwordEncoder() {
             "/Slide/**", "/TimeScript/**", "/OriginalScript/**", "/Brochure-English/**", "/Brochure/**",
             "/ResearchPaper/**", "/downloads", "/loadLanguageByCategoryResource",
             "/loadLanguageAndTopicByCategoryResource", "/loadTutorialCountByTopicAndLanguage",
-            "/loadTutorialCountByTopics", "/downloadResources", "/tutorialsToUploadScripts" };
+            "/loadTutorialCountByTopics", "/downloadResources", "/isTutorialPublishedandAddedQueue/**",
```

Comment:
is the isTutorialPublishedandAddedQueue url still there? if not, remove from here also.